namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-dev/resources/main.tf

```diff
@@ -33,3 +36,34 @@ resource "kubernetes_secret" "rds-instance" {
     secret_access_key   = "${module.rds-instance.secret_access_key}"
   }
 }
+##################################################
+
+########################################################
+# Publisher Elasticache Redis (for resque + job logging)
+module "publisher-elasticache" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=1.0"
+
+  ec_engine                   = "redis"
+  number_of_nodes             = 1
+
+  cluster_name                = "${var.cluster_name}"
+  cluster_state_bucket        = "${var.cluster_state_bucket}"
+
+  application                 = "formbuilderpublisher"
+  environment-name            = "${var.environment-name}"
+  is-production               = "${var.is_production}"
+  infrastructure-support      = "${var.infrastructure-support}"
+  team_name                   = "${var.team_name}"
+}
+
+resource "kubernetes_secret" "publisher-elasticache" {
+  metadata {
+    name      = "elasticache-formbuilder-publisher-dev"
+    namespace = "formbuilder-platform-dev"
+  }
+
+  data {
+    endpoint          = "${module.publisher-elasticache.endpoint}"
```

Comment:
`endpoint` is not an output of the module.
You should instead use `cache_nodes`, however, it's a map so you can't put it in a secret as is. For a single-node EC instance, you can use something like [this](https://github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster/blob/master/example/outputs.tf#L7) or set them as separate fields in the secret.

At some point during the changes, we did have `endpoint` as an output, perhaps we missed some part in the docs when updating?

Reply:
I probably either misread it, or was working off my own memory of a previous version. It's been quite a day today... :)
Fix pushed